src/config/entities/configuration.ts

```diff
@@ -131,11 +131,13 @@ export default () => ({
       apiKey: process.env.INFURA_API_KEY,
     },
   },
+  typeorm: { autoLoadEntities: true, manualInitialization: true },
```

Comment:
Nit: I've move these under the `db` property.

Reply:
In my opinion, a better approach than the current one would be to use `db.orm` and `Array<db.databases>`, allowing us the flexibility to add multiple databases in the future and easily iterate over them. Do you believe we should move Typeorm under db?

Reply:
`db.orm` sounds good to me. I wouldn't create and array out of them for now though as we don't have more than one. What do you think?

Reply:
`db.orm` also sounds good to me. Regarding the array, no strong opinion, but I'd slightly prefer not having it until it's needed. It can be easily added later if we need to add another db.

Reply:
@iamacook @hectorgomezv What do you think about moving `orm` under `db` and placing `postgres` under `db.connection`? My concern is that having `orm` and `postgres` under the same parent could lead to ambiguity. For now, `db.connection` will remain an object, but in the future, we can convert it to an array and rename it to `db.connections`(add an **s** to make it plural).

Reply:
Sounds good to me @PooyaRaki, thanks! 👍